src/Utils.ts

```diff
@@ -34,3 +34,8 @@ export function normalizePathForRemote(filePath: string) {
 	// remote debugger expects forward slashes on all platforms
 	return filePath.replace(/\\/g,"/");
 }
+
+export function isUUID(uuid: string) {
+	let regex = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$/i;
```

Comment:
Nit non-blocking, const
```suggestion
	const regex = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$/i;
```